components/molecules/MobileNavbarRightDiv.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
Let's avoid including references like `Div` in the reference. As soon as you apply some CSS on it, the behaviour is different from a regular div. (Remember: all the dom elements are basically the same, until CSS is applied on them.)